Course/IOS15(P1)/Lesson 1/Lesson1VIew.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Lesson1VIew.swift
```

Comment:
Mistake in naming, also call LessonView1 if you want of FirstLessonView. Also don't use `spaces`,  `()` in naming folders or files.

Reply:
got it!